package.json

```diff
@@ -38,12 +39,13 @@
   "engines": {
     "node": ">=8.0.0"
   },
+  "homepage": "https://github.com/mozilla/something-awesome/",
```

Comment:
Added this because I'm using it for the `/__version__` endpoint's `source` field.